server/jvm/build.gradle.kts

```diff
@@ -13,7 +13,6 @@ subprojects  {
     val junitVersion = "5.10.0"
     dependencies {
         implementation(platform("global.genesis:genesis-bom:${properties["genesisVersion"]}"))
-        testImplementation(kotlin("test"))
```

Comment:
is there a reason for this removal? It is definitely not mandatory, but it can also be useful to have.

Reply:
it's part of the migration guide to 7.1.0, leaving it in results in the following error:

![image](https://github.com/genesiscommunitysuccess/blank-app-seed/assets/101820611/8fe85d87-4f2c-49df-9f4b-83a80d6807d1)

Reply:
You can find more details (here)[https://genesisglobal.atlassian.net/browse/GSF-6631]